engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/DefinitionExtractor.scala

```diff
@@ -138,10 +138,11 @@ object DefinitionExtractor {
     }
 
     private def extractTypesFromObjectDefinition(obj: ObjectWithMethodDef): List[TypingResult] = {
-      def clazzRefFromTyped(typed: TypingResult): Iterable[ClazzRef] = typed match {
-        case Typed(possibleTypes) => possibleTypes.map(t => ClazzRef(t.klass))
-        case _ => Set()
-      }
+      //TODO: uncomment + fix when .flatMap(clazzRefFromTyped) will be uncommented
```

Comment:
maybe let's just remove this and the comment :)

Reply:
done